src/misc/ss.h

```diff
@@ -0,0 +1,25 @@
+//
+//  SS.hpp
+//  color_picker
+//
+//  Created by will on 2/3/19.
+//
+
+#include <QWidget>
+
+class SS : public QObject
+{
+public:
+    static QString QPushButtonInvisible();
+    static QString QPushButtonGreyscale();
+    static QString QPushButtonRounded(int size);
+    static QString QSpinBox();
+    static QString QSlider();
+    static QString QLineEdit();
+    static QString QWidgetDark();
+    static QString QLabelWhite();
+    static QString QComboBox();
+    static QString QPushButtonGrouped();
```

Comment:
I would much prefer prefixing these with something that denotes that its custom or naming them something else that represents a style instead of shadowing Qt's widget names.

Reply:
this was done to avoid ambiguity when searching for a particular style.